src/subcommand/init.rs

```diff
@@ -36,20 +38,24 @@ impl Init {
         let stdout = io::stdout();
         let mut handle = stdout.lock();
 
-        writeln!(handle, "{}", config.z).unwrap();
+        writeln!(handle, "{}", config.z)?;
```

Comment:
I think this should be left as `unwrap`. All the errors we return have a user-friendly `context` attached to them, but it doesn't seem useful to do that here.

Rather, if writing to `stdout` fails, we can (reasonably) assume the user's computer is on fire and panic.